testing/run-scripts/run_cloud.sh

```diff
@@ -154,24 +154,19 @@ HOST_IP=`ip -o -4 addr list docker0 | awk '{print $4}' | cut -d/ -f1`
 # Start Zork, if necessary.
 if ! docker ps -a | grep uproxy-zork >/dev/null; then
   if ! docker images | grep uproxy/$1 >/dev/null; then
-    BROWSER=$(echo $1 | cut -d - -f 1)
-    VERSION=$(echo $1 | cut -d - -f 2)
-    IMAGEARGS=
-    if [ -n "$PREBUILT" ]
+    if [ -n "$IMAGE" ]
     then
-      IMAGEARGS="-p"
+      IMAGE="soycode/uproxy-zork"
```

Comment:
Why don't you define the default value when you declare `$IMAGE`, right at the top?